src/routes/userRoutes.js

```diff
@@ -17,6 +18,7 @@ userRoutes.get('/api/v1/users', isLoggedIn, getAllUsers);
 userRoutes.post('/api/v1/users/signin', validator(userSchema), loginUser);
 userRoutes.post('/api/v1/users/signup', validator(userSchema), createNewUser);
 userRoutes.put('/api/v1/users/:id/roles', verifyRole('admin'), setRoles);
+userRoutes.put('/api/v1/users/:id/status', verifyRole('admin'), disableAccount);
```

Comment:
use patch because you going to update column and change the endpoint `/api/vi/users/:id/disable`